apps/token-manager/app/src/App.js

```diff
@@ -169,15 +168,20 @@ export default observe(
           appStateReady,
         }
       }
-      const { tokenSupply, holders, tokenDecimals } = state
+      const { holders, tokenDecimals, tokenSupply } = state
       const tokenDecimalsBase = new BN(10).pow(new BN(tokenDecimals))
       return {
         ...state,
         appStateReady,
         tokenDecimalsBase,
+        numData: {
```

Comment:
Maybe we should add a comment here, to say that these numbers are not safe integers?